wireapp/wire-android
# Feature - OnBoarding Flow tweaks - part 1

## What's new in this PR?

Added an informative screen to the custom domain onboarding flow.

https://wearezeta.atlassian.net/browse/AN-6890

### Testing

- Go to the welcome screen
- Press **Enterprise login**
- Type an email ending with **@qa-demo.com** 
- Press **OK**
- You should see an informative screen while switching to a custom backend. This screen is displayed for 5 seconds and disappears without user interaction.

## Screenshot

![device-2020-05-08-005720](https://user-images.githubusercontent.com/43655743/81353071-6e809180-90c8-11ea-8205-9c213dd59f65.png)

#### APK
[Download build #2070](http://10.10.124.11:8080/job/Pull%20Request%20Builder/2070/artifact/build/artifact/wire-dev-PR2832-2070.apk)
[Download build #2074](http://10.10.124.11:8080/job/Pull%20Request%20Builder/2074/artifact/build/artifact/wire-dev-PR2832-2074.apk)
